tests/test_justice_data_source.py

```diff
@@ -14,34 +16,83 @@ def test_host_port_parsing():
     ]
 
     for example in examples:
-        config_dict = {"base_url": example}
+        config_dict = {"base_url": example, "default_owner_email": default_owner_email}
         config = JusticeDataAPIConfig.parse_obj(config_dict)
         assert config.base_url == example
 
 
-def test_ingest():
-    source = JusticeDataAPISource(
-        ctx=PipelineContext(run_id="justice-api-source-test"),
-        config=JusticeDataAPIConfig(base_url="https://data.justice.gov.uk/api"),
-    )
+@pytest.fixture()
+def mock_ingest(default_owner_email):
```

Comment:
This name is a bit confusing - it does run the ingest, it's just the Justice data API that is mocked

Reply:
yeah agree, will change